src/UserCook/action.js

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+export const USER_SIGNED_UP = "user/SIGNED_UP";
+export const USERPROFILE_SIGNED_UP = "userProfile/SIGNED_UP";
+export const USERADDRESS_SIGNED_UP = "userAddress/SIGNED_UP";
+
+const baseUrl = "http://localhost:5000";
+
+// signup - table: user
+export function userSignUp(userData) {
+  return async (dispatch, getState) => {
+    try {
+      const responseData = await axios.post(`${baseUrl}/user/signup`, userData);
+
+      dispatch(userSignUp_Sucess(responseData.data));
+    } catch (error) {
+      throw error;
+    }
+  };
+}
+
+function userSignUp_Sucess(respData) {
+  return {
+    type: USER_SIGNED_UP,
+    payload: respData
+  };
+}
+
+// signup - table: user_profile
+export function userProfileSignUp(userProfileData) {
+  return async (dispatch, getState) => {
+    try {
+      const responseData = await axios.post(
+        `${baseUrl}/userprofile/signup`,
+        userProfileData
+      );
+
+      dispatch(userProfileSignUp_Sucess(responseData.data));
```

Comment:
if you are using camelCase for function and variables (which is the way to go) don't use underscores `_`

Reply:
thanks for the tip. will update and have it implemented by the next push.